File: JavaScript/8-kyu/Beginner - Lost Without a Map.js
// Given an array of integers, return a new array with each value doubled.

// For example:

// [1, 2, 3] --> [2, 4, 6]

function maps(x){
    let newArray = [];
    x.forEach(element => {
        newArray.push(element*2)
    });
    return newArray;
}

console.log(maps([1, 2, 3]))

File: Python/8-kyu/Beginner - Lost Without a Map.py
# DESCRIPTION:
# Given an array of integers, return a new array with each value doubled.
#
# For example:
#
# [1, 2, 3] --> [2, 4, 6]
def maps(a):
    new_list = []
    for i in a:
        i*=2
        new_list.append(i)
    return new_list
# print(maps([1, 2, 2]))